Modix/Modules/NoteModule.cs

```diff
@@ -0,0 +1,172 @@
+                {
+                    Log.Warning($"User: {Context.Message.Author.Username} has tried to create a note without a message");
+                    await ReplyAsync("Please ensure that you have included a message with your note");
+                }
+
+                await _moderationService.CreateInfractionAsync(InfractionType.Notice, guildUser.Id, message, null);
+
+                await ReplyAsync($"Note has been created for user: {guildUser.Username}");
+            }
+            catch (Exception e)
+            {
+                Log.Error(e, $"NoteModule CreateNote failed with the following user: {guildUser.Username} and message: {message}");
+                await ReplyAsync("Uh oh. Looks like you are using the command wrong or an error has occured.");
+            }
+        }
+
+        [Command("note remove"), Summary("Removes a note from a user")]
+        public async Task RemoveNote(long noteId)
+        {
+            var user = Context.User as SocketGuildUser;
+
+            // TODO: We shouldn't need to do this once claims are working
+            if (!IsStaff(user) && !IsOperator(user))
+            {
+                await ReplyAsync($"I'm sorry, @{user.Nickname}, I'm afraid I can't do that.");
+                return;
+            }
+
+            try
+            {
+                await _moderationService.RescindInfractionAsync(noteId, null);
+
+                await ReplyAsync($"Note {noteId} has been removed");
+            }
+            catch (Exception e)
+            {
+                Log.Error(e, $"NoteModule RemoveNote failed with the following noteId: {noteId}");
+                await ReplyAsync($"Failed to remove the note: {noteId}");
+            }
+        }
+
+        [Command("note search"), Summary("Search notes for a user")]
+        public async Task SearchNotesByUserId(ulong userId)
+        {
+            var user = Context.User as SocketGuildUser;
+
+            // TODO: We shouldn't need to do this once claims are working
+            if (!IsStaff(user) && !IsOperator(user))
+            {
+                await ReplyAsync($"I'm sorry, @{user.Nickname}, I'm afraid I can't do that.");
+                return;
+            }
+
+            try
+            {
+                var notes = await _moderationService.SearchInfractionsAsync(
+                    new InfractionSearchCriteria
+                    {
+                        SubjectId = (long)userId,
+                        Types = new[] { InfractionType.Notice }
+                    },
+                    new[]
+                    {
+                        new SortingCriteria { PropertyName = "CreateAction.Created", Direction = SortDirection.Descending }
+                    });
+
+                var sb = new StringBuilder();
+                var hints = new Hints { MaxTableWidth = 100 };
+                var formatter = new TableFormatter(hints);
+                sb.AppendLine("```");
+
```

Comment:
Discord's lib has a Format.Code() helper to automagically enclose your text in backticks

Reply:
Fixed in 598b769.